pkg/install/v1alpha1/cstor_volume.go

```diff
@@ -621,6 +621,10 @@ spec:
     Calculate the replica count
     Add as many poolUid to resources as there is replica count
     */}}
+    {{- $poolUids := keys .ListItems.cvolPoolList.pools }}
+    {{- $lblSelector := ifNotNil $appUniqVal "openebs.io/replica-anti-affinity: $appUniqVal" }}
```

Comment:
ifNotNil is not implemented till now.

Reply:
Implement or ReUse the Runtask that does a get of PVC.
Extract the label with key `openebs.io/replica-anti-affinity` & save it & then use it here as `$replicaAntiAffinityVal`
& then write below logic:
`{{- $lblSelector := printf "openebs.io/replica-anti-affinity: " $replicaAntiAffinityVal | ifNotNil $replicaAntiAffinityVal }}`

Above logic tries to get the entire labelSelector if available or blank if not available. You may try better logic if possible.